internal/controller/datadogagent/feature/factory.go

```diff
@@ -44,14 +46,12 @@ func BuildFeatures(dda *v2alpha1.DatadogAgent, options *Options) ([]Feature, []F
 	for key := range featureBuilders {
 		sortedkeys = append(sortedkeys, key)
 	}
-	sort.Slice(sortedkeys, func(i, j int) bool {
-		return sortedkeys[i] < sortedkeys[j]
-	})
+	slices.Sort(sortedkeys)
```

Comment:
This achieves the same thing, more efficiently, is the recommend way to do it: e.g. https://thekoreanguy.medium.com/which-function-should-you-use-to-sort-a-slice-faeb181f81af